media_service/iiif.py

```diff
@@ -240,7 +245,7 @@ def __init__(self, manifest, canvas_id):
         self.resource = None
 
     def add_image(self, image):
-        self.resource = IIIFImageResource(self.manifest, image['id'], image['url'], image['is_link'])
+        self.resource = IIIFImageResource(self.manifest, self.id, image['url'], image['is_link'])
```

Comment:
So as I understand it, this means that for `media_management_api` only one image resource per canvas is supported?

Also, am I reading [the specs](http://iiif.io/api/presentation/2.0/#image-resources) correctly here:
> Annotations may have their own URIs, conveyed by adding an @id property to the JSON object, and if so should be HTTP URIs. The content of the annotation should be returned if the URI is dereferenced. Annotations may be dereferenced separately from their annotation lists, sequences and manifests; some systems may do this and identifiers should be given using the recommended pattern if possible.
...in assuming that using the same `id` as the parent canvas, since the `build_url()` of the image resource has a different `object_type`, will evaluate to a URI that is distinguished from the canvas with the same `id` so that it can be dereferenced and simply return an image?